src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int side;
+    private int base;
+
+    public Rectangle(int side, int base, String color) {
+        this.side = side;
+        this.base = base;
+        this.setColor(color);
+    }
+
+    public int getSide() {
+        return side;
+    }
+
+    public int getBase() {
+        return base;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Drawing a rectangle");
+    }
+
+    @Override
+    public double getArea() {
+        return side * base;
+    }
+
+    @Override
+    public String getFigureData() {
```

Comment:
Use 'name()' instead of 'toString()' for getting the String representation of enum constants. If 'getColor()' is returning an enum, make sure it uses 'name()' internally.